documentation/asciidoc/accessories/iqaudio/configuration.adoc

```diff
@@ -18,66 +18,59 @@ Find the `dtparam=audio=on` line in the file and comment it out by placing a # s
 #dtparam=audio=on
 ----
 
-Press CTRL + X, then Y and ENTER to save, followed by a reboot of your Raspberry Pi in order for the settings to take effect:
+Press CTRL+X, then Y and Enter to save, followed by a reboot of your Raspberry Pi in order for the settings to take effect.
 
 ----
 $ sudo reboot
 ----
 
-Alternatively, the `/boot/config.txt` file can be edited directly onto the Raspberry Pi microSD card inserted into your usual computer. Using the default file manager open the `boot` volume on the card and edit the `config.txt` file using an appropriate text editor, such as Notepad or Notepad++, then save the file, eject the microSD card and reinsert it back into your Raspberry Pi.
+Alternatively, the `/boot/config.txt` file can be edited directly onto the Raspberry Pi's microSD card, inserted into your usual computer. Using the default file manager, open the `boot` volume on the card and edit the `config.txt` file using an appropriate text editor, such as Notepad or Notepad++, then save the file, eject the microSD card and reinsert it back into your Raspberry Pi.
```

Comment:
Not sure if we should recommend Notepad, as IIRC that saves files with DOS line-endings rather than Unix line-endings, and that might cause problems?

Reply:
This. We should not in any way recommend Notepad (or any other editor come to that).

Reply:
Fixed and chastened.